### ISSUE TITLE
Improve blobfuse option specific commandline documentation 

### ISSUE THREAD

**Issue by Tatsinnit** [Azure/azure-storage-fuse#286]
### Which version of the blobfuse was used?

Latest

### Which OS (please include version) are you using?
Ubuntu

### What problem was encountered?
if user pass single `-o` followed by multiple options like `-o {option} {option1} {option2}`

### Have you found a mitigation/solution?
so essentially like recommeded in your samples here : https://github.com/Azure/azure-storage-fuse/blob/master/mount.sh the right way is like : `o attr_timeout=240 -o entry_timeout=240 -o negative_timeout=120`

**Comment by amnguye** [comment]
Hi,

Can you point to me where in the documentation do we say for the user to say -o {option} {option1} {option2}. 

The wiki, readme and how to mount blob storage using blobfuse page and all of them are consistent with the mount.sh file. We provide examples on how to write the mount command that are clear in the documentation.

**Comment by Tatsinnit** [comment]
Cool, thanks for reply, Ah, let me rephrase as to what I think will be better because I am being asked few times by the folks who are using this **or** might be if you think it is not necessary this Issue will be good for its clarification anyways. :)  

:bulb: I think the confusion for some reader could or at least I was asked few times by folks is: that then does `-o {opt1} {opt2}` is supported? so essentially if we make it clear some where then that will add clarity to the docs, rightnow we ahve a sample and it says this is how it works. No biggie, but if you are happy to keep this issue as clarity anyways then feel free to set it as closed and we take that the legit option is the one mentioned int the samples.

**Comment by amnguye** [comment]
I don't think it's an issue if in all our documentation we have examples and instructions says to put a -o in front of the option. I can't find in our documentation where this has been unclear. Also these are the instructions/options by FUSE not blobfuse.
